packages/by-name/kata/runtime-class-files/package.nix

```diff
@@ -0,0 +1,43 @@
+# Copyright 2024 Edgeless Systems GmbH
+# SPDX-License-Identifier: AGPL-3.0-only
+
+{ stdenvNoCC
+, kata
+, fetchurl
+, OVMF
+}:
+
+let
+  image = kata.kata-image;
+  kernel = "${kata.kata-kernel-uvm}/bzImage";
+
+  # TODO(msanft): building a non-NixOS QEMU is hard, investigate later and pin it for now.
+  qemu-bin = fetchurl {
+    url = "https://cdn.confidential.cloud/contrast/node-components/1718800762/qemu-system-x86_64";
+    sha256 = "sha256-7MS/tK6q4D8y/FH6VcfARQLhIuvtNP6TsGfy+0o9kSc=";
+  };
```

Comment:
In the guest, we solve this by copying the closure into the image. In theory, we could do the same on the host, right?

Reply:
Agreed in person to do this in a follow-up PR